### ISSUE TITLE
Rename suggestions provider and enable it again

### ISSUE THREAD

**Issue by dschuermann** [Free-Software-for-Android/Standalone-Calendar#3]
see AndroidManifest.

it should not share the same namespace as "com.android.calendar" to prevent conflicting with already installed calendars

**Comment by dalb8** [comment]
I just installed this on a Gapps device and this was the obvious thing I saw (that the Google Calendar was already filled). Comparing with the Google Calendar pre-installed it was hard to tell the difference. The About section is blank and the icon is different. I was reading a lot of commentary last week about the death of AOSP at the hands of Google Play Services; it doesn't seem to be evident here!

**Comment by dschuermann** [comment]
Right. I started standalone calendar just beacause I read those articles that the AOSP calendar will not be developed any more. I was also suprised, but it looks like those things are missing (but I haven't investigated further, I am not using Google's services normally):
- no sync of notifications (this is done via GCM I think), so if you read one notification on one device it is also read on another device
- no color changing option for calendars (in Google's calendar this can be done for calendars synced with google account)

**Comment by dalb8** [comment]
- Yes though the issue really is how easy it will be add notification syncing for other systems like CalDav
- I can change calendar colours in Standalone Calendar and the colours are synced to Google Calendar app so it must be dependent on Gapps

**Comment by dschuermann** [comment]
Thanks for testing the second point. Then there is really not that much difference between Standalone Calendar and Google Calendar app. I think notification sync is also not really part of the calendar than part of the sync service.

**Comment by dschuermann** [comment]
Looks like the reddit guy published the code at https://github.com/bvermeule/Launcher3

Unfortunately I don't have time to look into the launcher in more detail, it's not on my personal priority list ;)

**Comment by dalb8** [comment]
Yeah, I saw that later (of course). All the other forks (including Cyanogenmod) do nothing more but remap the settings button so far, but it's interesting all the same how they try to bring it under the SDK

**Comment by dalb8** [comment]
Sorry if off-topic, but I've been looking at AOSP and see that one app, Pinyin IME, could be easy to build with the SDK. If anybody could provide a pointer that would be appreciated.
